client/blocks/comments/post-comment.jsx

```diff
@@ -477,6 +497,12 @@ class PostComment extends PureComponent {
 					/>
 				) }
 				{ this.renderRepliesList() }
+				{ config.isEnabled( 'reader/login-window' ) && (
+					<ReaderJoinConversationDialog
+						onClose={ this.hideJoinConversationModal }
+						isVisible={ this.state.showJoinConversationModal }
+					/>
```

Comment:
I notice we are adding this component in a few different places (inside likes, comments, etc.).

Im wondering if we should set this component at a higher level in the dom/componentry so it only needs 1 instantiation for all the places that might want to open it. Then comments, likes, and every other action we may want to add this to just needs to call a function to open that higher level master version of the component and not have to add a new child version in each file. Similarly I think this would also save us from needing to redefine `onLoggedOut` the same way in various places.

Reply:
Good idea. I'll look into this approach.